{{cookiecutter.python_name}}/package.json

```diff
@@ -12,7 +12,10 @@
     "url": "{{ cookiecutter.repository }}/issues"
   },
   "license": "BSD-3-Clause",
-  "author": "{{ cookiecutter.author_name }}",
+  "author": {
+    "name": "{{ cookiecutter.author_name }}",
+    "email": "{{ cookiecutter.author_email }}"
+  },
```

Comment:
According to the [npm documentation](https://docs.npmjs.com/cli/v6/configuring-npm/package-json#people-fields-author-contributors), `author` is not a string but an object with up to three properties: `name`, `email` and `url`.